release/src/main/scripts/build_release_candidate.sh

```diff
@@ -228,20 +229,14 @@ if [[ $confirmation = "y" ]]; then
   cd ~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_WEBSITE_REPO}
```

Comment:
`Updating Beam Website` would be more accurately `Updating Release Docs`

Reply:
Done